clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -5192,15 +4984,13 @@ bool SYCLIntegrationFooter::emit(raw_ostream &OS) {
 // -----------------------------------------------------------------------------
 // Utility class methods
 // -----------------------------------------------------------------------------
-
-bool Util::isSyclAccessorType(QualType Ty) {
-  return isSyclType(Ty, "accessor", true /*Tmpl*/);
+bool Util::isSyclSpecialType(const QualType Ty) {
```

Comment:
Sorry I just noticed this. Is there a reason you are checking for `__init `instead of checking for the attribute?